app/lib/frontend/templates/views/layout.mustache

```diff
@@ -148,5 +148,11 @@
 {{& schema_org_searchaction_json}}
 </script>
 {{/schema_org_searchaction_json}}
+
+{{#page_data_json}}
+<script type="application/ld+json">
+{{& page_data_json}}
+</script>
+{{/page_data_json}}
```

Comment:
Interesting... I'm curious, do you imagine we'll ship a lot of data this way?

How about just using data properties, `<div data-package-name='...' data-package-version='...'>` on some element we already have...

Reply:
`<div>` could work, but I think it is more future-proof to define a JSON structure on both ends and do the encoding/decoding. For example this way we can have embedded maps and lists, and we don't need to wrangle with attribute names how they are mapped to lists.